rewrite-maven/src/test/java/org/openrewrite/maven/internal/MavenPomDownloaderTest.java

```diff
@@ -573,6 +574,42 @@ void skipsLocalInvalidArtifactsEmptyJar(@TempDir Path localRepository) throws IO
                 .download(new GroupArtifactVersion("com.bad", "bad-artifact", "1"), null, null, List.of(mavenLocal)));
         }
 
+        @Test
+        @SetSystemProperty(key = "org.openrewrite.allowPomDownloadFailure", value = "true")
+        void allowPomDowloadFailure(@TempDir Path localRepository) throws IOException, MavenDownloadingException {
+            Path localArtifact = localRepository.resolve("com/some/some-artifact");
+            assertThat(localArtifact.toFile().mkdirs()).isTrue();
+            Files.createDirectories(localArtifact.resolve("1"));
+
+            Path localPom = localRepository.resolve("com/some/some-artifact/1/some-artifact-1.pom");
+            Files.writeString(localPom,
+              //language=xml
+              """
+                 <project>
+                   <groupId>com.some</groupId>
+                   <artifactId>some-artifact</artifactId>
+                   <version>1</version>
+                 </project>
+                """
+            );
+            Path localJar = localRepository.resolve("com/some/some-artifact/1/some-artifact-1.jar");
+            Files.writeString(localJar, "");
+
```

Comment:
It looks like none of these lines are necessary to make the test pass; Is there any reason to keep these then? Should the logic reflect perhaps that we're ok to continue if only the `.jar` is present?
```suggestion
```